scripts/upload-gh-pages.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+git config --global user.name "Travis CI"
+git config --global user.email "noreply+travis@fossasia.org"
+
+sudo pip install Sphinx
+
+cd docs/sources
+
+make html
+
+git clone --quiet --branch=gh-pages https://fossasia:$GITHUB_KEY@github.com/fossasia/phimpme-android gh-pages > /dev/null
+cd gh-pages
+
+rm -rf *
```

Comment:
empty the hidden files on gh-pages excluding .git. because rm -rf wont clear those

Reply:
Should i use rm -rf .*

Reply:
https://github.com/fossasia/phimpme-android/pull/2854